package.json

```diff
@@ -8,17 +8,19 @@
   "packageManager": "yarn@3.5.0",
   "private": true,
   "engines": {
-    "node": "^18"
+    "node": "^16 || ^18"
```

Comment:
this prevents yarn from failing when using Node 16. (which you have to do in order to build `wc-nuxt2` from root